lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -209,6 +215,14 @@ def __init__(
         self.distance_to_surface = distance_to_surface
 
         if hydrodynamically_correct:
+            if self.axial:
+                raise NotImplementedError(
+                    "No hydrodynamically correct axial force model is currently available."
+                )
+
+            # The hydrodynamically correct model already accounts for the effect of a nearby wall.
+            # Here the drag coefficient in the model represents the bulk drag coefficient.
+            self._drag_correction_factor = 1
             # This model is only valid up to l/R < 1.5 [6] so throw in case that is violated.
             if distance_to_surface and distance_to_surface / (self.bead_diameter / 2) < 1.5:
                 raise ValueError(
```

Comment:
Not really related to this PR, but I noticed during testing that this message could be a little bit clearer:
It raises
``` 
"This model is only valid for distances to the surface ..."
```
but it could be something like
```
"A hydrodynamically correct model is only valid for distances to the surface ..."
```

In this way, the user can figure out that hydro can be disabled if they are working so close to the surface.

Reply:
The error message could also include explicit troubleshooting advice: "If you are working very close to the surface, turn off the hydrodynamic correction."

Reply:
Good ideas. I've combined the two suggestions into:
```python
                    "The hydrodynamically correct model is only valid for distances to the surface "
                    "larger than 1.5 times the bead radius. For distances closer to the surface, "
                    "turn off the hydrodynamically correct model."
```